### PR TITLE
Hides the preemptible checkbox for CoreWeave

### PR DESCRIPTION
**Pull request by jesseconductor**
NOTE: Ignore the last commit (linting) to make reviewing changes easier.

## Summary

Hides Spot checkbox when CW is the provider. Fixes some Python 2.7 compatibility issues.

## Type of change

- [ ] Non-functional
- [ ] Fix
- [x ] Feature
- [ ] Breaking change

## Checklist before requesting a review

- [ ] My code follows the style guidelines of this project
- [ ] I have refactored to remove duplication
- [ ] I have added comments to unidiomatic code (hint: there shouldn't be any)
- [ ] I have performed a self-review
- [ ] I have made corresponding changes to the documentation
- [ ] I have updated compatibility information in the README
- [ ] My changes generate no new warnings
- [ ] I have added tests
- [ ] New and existing tests pass locally
- [ ] Changes to dependencies have been merged and published

## Note to reviewers

Please use codes in comments to indicate whether or not action should be taken. For example:
* `[NIT]` Spelling, naming, minor preference, or style issue.
* `[OPT]` Code issue, far from critical. Optional to fix.
* `[REQ]` Requires fixing due to guidelines or critical bugs.

### CODE REVIEW

File: custom/scripts/Jobs/send_to_conductor.py [hunk 1]
```diff
@@ -221,7 +221,7 @@ def onOKButtonClicked(self):
             self.conductorJob.instance_type = self.selectedInstanceType
             self.conductorJob.instance_count = self.deadlineJob.TaskCount
             self.conductorJob.job_title = self.jobNameTextBox.text()            
-            self.conductorJob.preemptible = self.preemptibleCheckBox.isChecked()       
+            self.conductorJob.preemptible = (self.cloud_provider != "CW" and self.preemptibleCheckBox.isChecked())
```
Comment (flebel) [inline custom/scripts/Jobs/send_to_conductor.py]: `CW` needs to be a constant.

File: custom/scripts/Jobs/send_to_conductor.py [hunk 2]
```diff
@@ -303,8 +303,13 @@ def getInstances(self):
 
         instances = [i for i in tree_data.instance_types.values() if i['operating_system'] == 'linux']
         instances = sorted(instances, key=operator.itemgetter("cores", "memory"), reverse=False)
+
+        #self.cloud_provider = instances.get_cloud_provider()
+        self.cloud_provider = "CW"
```
Comment (flebel) [inline custom/scripts/Jobs/send_to_conductor.py]: `CW` needs to be a constant.

File: custom/scripts/Jobs/send_to_conductor.py [hunk 3]
```diff
@@ -304,10 +303,8 @@ def getInstances(self):
         instances = [i for i in tree_data.instance_types.values() if i['operating_system'] == 'linux']
         instances = sorted(instances, key=operator.itemgetter("cores", "memory"), reverse=False)
 
-        #self.cloud_provider = instances.get_cloud_provider()
-        self.cloud_provider = "CW"
-
-        self.preemptibleCheckBox.setVisible(self.cloud_provider != "CW")
+        self.cloud_provider = tree_data.provider 
+        self.spotCheckBox.setVisible(self.cloud_provider != "cw")
```
Comment (flebel) [inline custom/scripts/Jobs/send_to_conductor.py]: Please use a constant for `cw`.

**Review by flebel — APPROVED** [review]

File: custom/scripts/Jobs/send_to_conductor.py (hunk 1 above)
Comment (jesseconductor) [reply custom/scripts/Jobs/send_to_conductor.py]: I replaced with a method to improve readability

**Pull request merged**